docs/conf.py

```diff
@@ -56,7 +56,7 @@
 
 # General information about the project.
 project = "python-miio"
-copyright = "2017, Teemu Rytilahti"
+copyright = "2017-2022, Teemu Rytilahti"
```

Comment:
Did you forgot to add some files to your recent commit, considering this is the only doc change?

Reply:
At some point when there are proper interfaces for different device types, it would be great to have a note about them for downstream implementers who only wish to supported the devices using those interfaces. But that can wait, so it's fine.